docs/guide/http_cache.md

```diff
@@ -72,573 +80,752 @@ fos_http_cache:
+$responseTagger->addTags([ContentTagInterface::RELATION_PREFIX . '33', ContentTagInterface::RELATION_PREFIX . '44']);
+```
 
-    In case of complex content, for instance Landing Pages with many blocks, you might get into trouble with too long response `xkey` header. Because of this, necessary cache entries may not be tagged properly. You will also see `502 Headers too long` errors.
-    If this is the case, customize the following runtime settings on your Varnish instance(s):
+See [Tagging from code](https://foshttpcachebundle.readthedocs.io/en/2.8.0/features/tagging.html#tagging-and-invalidating-from-php-code) in FOSHttpCacheBundle doc.
 
-    - [http_resp_hdr_len](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-hdr-len) (e.g. 32k)
-    - [http_max_hdr](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-max-hdr) (e.g. 128)
-    - [http_resp_size](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-size) (e.g. 64k)
-    - [workspace_backend](https://varnish-cache.org/docs/6.0/reference/varnishd.html#workspace-backend) (e.g. 128k)
+4\. Use deprecated `X-Location-Id` header.
 
-    If you need to see these long headers in the `varnishlog` adapt the [vsl_reclen](https://varnish-cache.org/docs/5.1/reference/varnishd.html#vsl-reclen) setting.
+For custom or eZ controllers (e.g. REST) still using `X-Location-Id`, `XLocationIdResponseSubscriber` handles translating
+this header to tags for you. It supports singular and comma separated Location ID value(s):
 
-### Response tagging process
+```php
+/** @var \Symfony\Component\HttpFoundation\Response $response */
+$response->headers->set('X-Location-Id', 123);
 
-#### For Content View
+// Alternatively using several location id values
+$response->headers->set('X-Location-Id', '123,212,42');
+```
 
-For Content View there is a dedicated response listener `HttpCacheResponseSubscriber`
-that triggers a set of Response taggers responsible for translating info from the objects
-involved in generating the view to corresponding tags as listed above.
-These can be found in `ezplatform-http-cache/src/ResponseTagger`.
+!!! caution "X-Location-Id use is deprecated"
 
-#### For responses with X-Location-Id
+    `X-Location-Id` is deprecated and will be removed in future version. For rendering content it is advised to refactor
+    to use Content View, if not applicable `ContentTagInterface` or lastly manually output tags.
 
-For custom or eZ Platform controllers still using `X-Location-Id`, a dedicated response listener
-`XLocationIdResponseSubscriber` handles translating this to tags so the cache can be properly invalidated by this bundle.
 
-!!! note
+### Response tagging in Templates
 
-    This is currently marked as deprecated. For rendering content it is advised to refactor to use Content View.
-    For other needs there is an FOS tag handler for Twig and PHP that can be used.
+1\. `ez_http_tag_location()`
```

Comment:
Actually, the function is named [´ez_http_cache_tag_location`](https://github.com/ezsystems/ezplatform-http-cache/blob/master/src/Twig/ContentTaggingExtension.php#L40).

The nameing was changed [here](https://github.com/ezsystems/ezplatform-http-cache/commit/01e29e4c784bc3d17c7b7a4c9e67e284d3e99340) but neither developer-documentation or markdown files in ezplatform-http-cache was updated accordingly, 

The `ez_http_tag_relation_ids()` and `ez_http_tag_relation_location_ids()` functions were added later, but their nameing is not consistent. They should both be renamed with a `ez_http_cache` prefix, right ? @alongosz

Reply:
Ok, we'll change that in another PR together with some more proofing of the text.

Reply:
Ok, possible fix on product side in https://github.com/ezsystems/ezplatform-http-cache/pull/135